docs/design/admission_control_limit_range.md

```diff
@@ -0,0 +1,122 @@
+# Admission control plugin: LimitRanger
+
+## Background
+
+This document proposes a system for enforcing min/max limits per resource as part of admission control.
```

Comment:
100% agree, I think this document has lived a long time, so wanted to just get agreement on the design.  The implementation PRs can add the user documentation.